src/Sylius/Bundle/CartBundle/spec/EventListener/CartSubscriberSpec.php

```diff
@@ -102,29 +102,26 @@ function it_should_remove_a_item_to_a_cart_from_event(CartItemEvent $event, Cart
         $this->removeItem($event);
     }
 
-    function it_should_clear_a_cart_from_event(CartEvent $event, CartInterface $cart, $manager, $provider)
+    function it_should_clear_a_cart_from_event(CartEvent $event, CartInterface $cart, $manager)
```

Comment:
If you're changing this line, replace `$manager` with `ObjectManager $manager` (and put it as the first argument) :)